docs/content/reports/_index.md

```diff
@@ -10,4 +10,4 @@ pre = "<i class='fas fa-scroll'></i> "
 
 The Monkey offers three reports:
 
-{{% children %}}
+{{% children description="true" %}}
```

Comment:
Add `style="p"` here, to have a more proper format for the list.

Reply:
That ends up looking like this. Is this what we want?

![image](https://user-images.githubusercontent.com/19957806/103783166-5c177880-5006-11eb-84b5-3f38ce977f5a.png)